src/app/containers/DutyRosterSheriffCard.tsx

```diff
@@ -28,25 +32,50 @@ interface ConnectedDutyRosterSheriffCardStateProps {
 class ConnectedDutyRosterSheriffCard extends React.Component<ConnectedDutyRosterSheriffCardProps
     & ConnectedDutyRosterSheriffCardStateProps
     & ConnectedDutyRosterSheriffCardDispatchProps> {
-        
-    getShiftDisplayForDate(date: TimeType): string {
+
+    getShiftDisplayForDate(date: TimeType): { shiftTime: string, workSectionId?: WorkSectionCode } {
         const { shifts } = this.props;
         const shiftsForDay = shifts.filter(s => moment(date).isSame(s.startDateTime, 'day'));
         if (shiftsForDay.length > 0) {
-            return `${moment(shiftsForDay[0].startDateTime).format('HH:mm')} 
-                    - ${moment(shiftsForDay[0].endDateTime).format('HH:mm')}`;
-        } 
-        return '';
+            const { startDateTime, endDateTime, workSectionId } = shiftsForDay[0];
+            return {
+                shiftTime: `${moment(startDateTime).format('HH:mm')} 
+                    - ${moment(endDateTime).format('HH:mm')}`,
+                workSectionId: workSectionId
+            };
+        }
+        return { shiftTime: '', workSectionId: undefined };
     }
 
     render() {
         const { visibleTimeStart, sheriff } = this.props;
-        const shiftSummary = this.getShiftDisplayForDate(moment(visibleTimeStart).toISOString());
-        const isCardDisabled = shiftSummary === '';
+        const {shiftTime, workSectionId} = this.getShiftDisplayForDate(moment(visibleTimeStart).toISOString());
+        const isCardDisabled = shiftTime === '';
+        const foreground = getForegroundColor(getWorkSectionColour(workSectionId));
         return (
             <SheriffListCard sheriff={sheriff} disabled={isCardDisabled} >
-                {!isCardDisabled && <div style={{fontSize: 14}}>{shiftSummary}</div>}
-            </SheriffListCard> 
+                {!isCardDisabled &&
+                    <div
+                        style={{
+                            fontSize: 14,
+                            fontWeight: 'bold'
+                        }}
+                    >
+                        {shiftTime}
+                        <WorkSectionIndicator workSectionId={workSectionId} orientation={'bottom-right'}/>
+                        <div
```

Comment:
This is the Worksection letter over the top right?  Perhaps it could be added to the WorkSection indicator module and have a flag `showWorksectionLabel` or something